### ISSUE TITLE
Does origin COCO pretrain not make any improvement?

### ISSUE THREAD

**Issue by Lily19970207** [ucas-vg/TinyBenchmark#35]
Hello, I am very interested in your work. Did you pre-train a checkpoint on Coco and did not make any improvement on the network?

**Comment by yinglang** [comment]
To be honestly,  In the experiment we do before paper accepted , we found the COCO pre-train does not have any improvement. But In Lately experiment, we found it can make some improvement in Faster RCNN-FPN if we do not inherit the anchor setting of pre-train dataset, show as in [the section](https://github.com/ucas-vg/TinyBenchmark/tree/master/tiny_benchmark#21-scale-match-on-fpn-tinyperson). But in RetinaNet, we do not found any improvement, show as in [this section](https://github.com/ucas-vg/TinyBenchmark/tree/master/tiny_benchmark#22-scale-match-on-adaptive-retinanet-tinyperson) if you found any different conclusion, very welcome to have some discuss.
